flooder/src/config.rs

```diff
@@ -0,0 +1,39 @@
+use clap::Parser;
+use std::{fs, path::PathBuf};
+
+#[derive(Debug, Parser)]
+#[clap(version = "1.0")]
+pub struct Config {
+    /// URL address(es) of the nodes to send transactions to
+    #[clap(long, default_value = "127.0.0.1:9945g")]
+    pub nodes: Vec<String>,
+
+    /// how many transactions to send
+    #[clap(long, default_value = "10000")]
+    pub transactions: u64,
+
+    /// what throughput to use (transactions/s)
+    #[clap(long, default_value = "1000")]
+    pub throughput: u64,
+
+    /// how many threads to create
+    #[clap(long, default_value = "10")]
+    pub threads: u64,
```

Comment:
Correct me if I am wrong, but with rayon-rs the threads are no longer needed?

Reply:
Or alternatively, we can set threads number for rayon, but I think I would prefer the default nr to be chosen.

Reply:
done